src/workerd/io/worker.c++

```diff
@@ -1971,7 +1971,15 @@ kj::Maybe<kj::Own<api::ExportedHandler>> Worker::Lock::getExportedHandler(
 
   kj::StringPtr n = name.orDefault("default"_kj);
   KJ_IF_SOME(h, worker.impl->namedHandlers.find(n)) {
-    return fakeOwn(h);
+    kj::Own<api::ExportedHandler> ret = fakeOwn(h);
```

Comment:
Why declare this variable `ret` here when it's only used in one case below? Can the last line just be `return fakeOwn(h);`?